components/PresidentialResults/index.js

```diff
@@ -33,7 +33,12 @@ export default function PresidentialResults() {
     () => fetchResults(userAnswers.filter((answer) => answer.answerId)),
   );
   const results = response?.data;
-
+  const firstresults = results?.filter(
```

Comment:
Usamos camelCase, qué te parece top en vez de first?
```suggestion
  const topResults = results?.filter(
```

Reply:
claro, creo que sí tiene más sentido